### ISSUE TITLE
Error: Build serf container

### ISSUE THREAD

**Issue by rshirochenko** [pomadchin/geo-env#11]
I`ve got error while building serf container with command ./build.sh. The problem that /tmp/serf.zip https://dl.bintray.com/mitchellh/serf/0.5.0_linux_amd64.zip link is broken.

```
Step 6 : RUN curl -Lso /tmp/serf.zip https://dl.bintray.com/mitchellh/serf/0.5.0_linux_amd64.zip
 ---> Using cache
 ---> 51b12df9236e
Step 7 : RUN unzip /tmp/serf.zip -d /bin
 ---> Running in c13edb462f82
Archive:  /tmp/serf.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of /tmp/serf.zip or
        /tmp/serf.zip.zip, and cannot find /tmp/serf.zip.ZIP, period.
INFO[0000] The command [/bin/sh -c unzip /tmp/serf.zip -d /bin] returned a non-zero code: 9 

```

**Comment by pomadchin** [comment]
Fixed already: https://github.com/geotrellis/geodocker-cluster/blob/master/serf/Dockerfile#L23 by https://github.com/geotrellis/geodocker-cluster/commit/ea51001e8aa94f8643ee6c68b886949928c8305a
Probably you have to pull repo up to the master.

**Comment by pomadchin** [comment]
Sorry, yes there is such a problem, for some reason bintray removed links. Anyway you can pull this image from the dockerhub: `docker pull daunnc/serf`

**Comment by pomadchin** [comment]
closed #10
